packages/browser/test/unit/tracekit/chromium.test.ts

```diff
@@ -579,6 +579,7 @@ describe('Tracekit - Chrome Tests', () => {
       name: 'Error',
       stack: `Error: bad
           at something (http://localhost:5000/(some)/(thing)/index.html:20:16)
+          at http://localhost:5000/(group)/[route]/script.js:1:126
           at more (http://localhost:5000/(some)/(thing)/index.html:25:7)`,
```

Comment:
Is this could be passed?
```
at (http://localhost:5000/(group)/[route]/script.js:1:126)
```

Reply:
I had the same thought but chrome doesn't emit such frames I think.

Reply:
We haven't seen stack lines like that. They only appear to be surrounded in parenthesis when there is filename.